act-rules-format/act-rules-format.bs

```diff
@@ -433,6 +440,20 @@ Even concepts like headings and images can be misunderstood. These terms could r
   </blockquote>
 </aside>
 
+<aside class=example>
+  <header>Example applicability of a rule with visual headings as a test target</header>
+  <blockquote>
+    <p>The rule applies to any HTML element that is styled as a heading.</p>
+    <p>Since this applicability is subjective, it requires a glossary definition for "styled as a heading".</p>
+    <p>Styled as a heading defintion - An element is considered to be styled as a heading when it exhibits at least 2 of the following characteristics.</p>
+    <ul>
+      <li>Element text is visually distinct from nearby text, such as through the use of a larger font-size or bolding.</li>
+      <li>Element is clearly delineated from surrounding text, either through the use of spacing or other visual effects.</li>
+      <li>The element text broadly describes the content that follows it in the reading order of the page.</li>
```

Comment:
To avoid including unintended targets, how about making the third item `and` instead of `or`? So, (1 OR 2) AND 3.